pootle/core/helpers.py

```diff
@@ -84,10 +84,11 @@ def get_translation_context(request):
     vfolder_pk = getattr(request, 'current_vfolder', '')
     display_priority = False
 
-    if not vfolder_pk:
-        display_priority = VirtualFolder.objects.filter(
-            units__store__pootle_path__startswith=request.pootle_path
-        ).exists()
+    if not vfolder_pk and not request.pootle_path.startswith("/projects"):
```

Comment:
If we are implementing this optimization as a temporary speedup (until vfolders are displayed outside TP level), then you might want to also leave out requests with `pootle_path` for a specific language (So only calculate if `pootle_path` has at least two items and it doesn't start with `/projects/`, like `/lang/proj/` or `/lang/proj/whatever/`)